model/riscv_insts_zce_ext.sail

```diff
@@ -0,0 +1,1021 @@
+/*=======================================================================================*/
+/*  RISCV Sail Model                                                                     */
+/*                                                                                       */
+/*  Copyright (c) 2017-2022                                                              */
```

Comment:
Those years look somewhat wrong. Also should be one file per extension.

Reply:
Will change them, this is the only file for this extension.